frontend/public/components/replication-controller.jsx

```diff
@@ -35,7 +35,9 @@ const Details = ({obj: replicationController}) => {
           </ResourceSummary>
         </div>
         <div className="col-md-6">
-          <ResourcePodCount resource={replicationController} />
+          <dt>Phase</dt>
+          <dd>{ _.get(replicationController, ['metadata', 'annotations', 'openshift.io/deployment.phase']) }</dd>
+          <dd><ResourcePodCount resource={replicationController} /></dd>
```

Comment:
`ResourcePodCount` already adds the dt/dd elements

```suggestion
          <ResourcePodCount resource={replicationController} />
```